behavior/microservice/app.py

```diff
@@ -41,10 +42,122 @@ def infer(model_type, ou_type):
     return jsonify(Y)
 
 
+def connect():
+    # pylint: disable=assigning-non-slot
+    db = getattr(g, "__database", None)
```

Comment:
If you want to be super nitpicky, I think the convention used in Flask is single underscore `_database` (like a "please do not use, internals only" Python member). Unless you wanted the anti name mangling?

```suggestion
    db = getattr(g, "_database", None)
```